test/chainlinkAggregator.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Signer } from '@ethersproject/abstract-signer'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import hre from 'hardhat'
+import { ChainlinkAggregator } from '../types/typechain'
+import { getMarket } from '../tasks'
+import { Address } from '../types/custom/config-types'
+import { ETH_ADDRESS } from '../consts'
```

Comment:
for addresses we should stick to using the token addresses in the config instead of consts 
```typescript
import { getTokens } from '../config/tokens'
const tokens = getTokens(<Network>hre.network.name)
tokens.ETH
```

Reply:
done